backend/src/zango/apps/appauth/models.py

```diff
@@ -53,6 +53,8 @@ def delete(self, *args, **kwargs):
 
 
 class AppUserModel(AbstractZangoUserModel, PermissionMixin):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    slug = models.SlugField(unique=True, blank=True, null=True)
```

Comment:
Hey @Yadavanurag13 , thankyou for this PR. I have a small comment.
Id field is already there in the the `AbstractZangoUserModel` and we don't need to override it. We needed a field of type uuid to track app users across our application and its not easy to guess(unlike id field). The reason we named the field slug is because we are going to use it to identify a user. 
So, can you please make the below changes, create new migrations and push the latest changes?
```suggestion
    slug = models.UUIDField(default=uuid.uuid4, editable=False)
```